CRM/Contribute/Page/ContributionRecur.php

```diff
@@ -75,6 +75,34 @@ public function view() {
 
     $this->assign('recur', $contributionRecur);
 
+    $templateContribution = CRM_Contribute_BAO_ContributionRecur::getTemplateContribution($this->getEntityId());
+
+    $lineItems = [];
+    $displayLineItems = FALSE;
+    if (!empty($templateContribution['id'])) {
+      $lineItems = [CRM_Price_BAO_LineItem::getLineItemsByContributionID(($templateContribution['id']))];
+      $firstLineItem = reset($lineItems[0]);
+      if (empty($firstLineItem['price_set_id'])) {
```

Comment:
Did this turn out to be possible in your testing? I would have expected it to always be present

Reply:
Yes it turned out to be possible. I had two recurring contributions without a contribution linked.